### ISSUE TITLE
Language selector hard to read in dark mode

### ISSUE THREAD

**Issue by dhruvkb** [WordPress/openverse#5277]
## Description

When the theme is set to dark, the language selector options are hard to read. This is because the options are rendered in a light font on a white surface.

## Screenshots

![Image](https://github.com/user-attachments/assets/9bab27bf-e885-42c5-b572-08cba0bf5d65)

## Environment

- Device: desktop
- OS: Windows 11
- Browser: Chrome 131.0.6778.140

## Additional context

This was reported on [Slack](https://wordpress.slack.com/archives/C02012JB00N/p1734179506780939).

**Comment by dhruvkb** [comment]
Okay, this was really bothering me because the select field dropdown is supposed to be a component rendered by the operating system so it should definitely have nothing to do with the color scheme of the website. 

So I looked into this using BrowserStack and it turns out that on Windows (crucially different from the behaviour of macOS) the text color and background color of the `<select>` dropdown is actually determined from the individual `<option>` elements and can even be different for different options!

<img width="600" alt="Image" src="https://github.com/user-attachments/assets/f53967ca-8acc-4193-8eb4-77e568e68301" />
